### ISSUE TITLE
Include validator withdrawals in calendar

### ISSUE THREAD

**Issue by gnidan** [rotki/rotki#8395]
## Abstract

It'd be nice to see past and possibly even future predicted Ethereum validator withdrawals inside the calendar.

## Motivation

At a glance, I want to be able to see a rough schedule for the earnings of the validators I own/help operate. It'd be nice for this to include estimates moving forward, but I understand that might be a bit tricky to implement properly.

**Comment by LefterisJP** [comment]
hey @gnidan !

Thanks for the feature request.

So let me see if I understand correctly. can you describe the ask a bit in more detail?

Say you own 5 validators. You would like to see in the calendar when the expected withdrawal timestamp would be for each?

The problem I can think of with this is that it's going to be moving as this constantly depends on the number of active validators and that is a value going up and down.

**Comment by gnidan** [comment]
> Say you own 5 validators. You would like to see in the calendar when the expected withdrawal timestamp would be for each?

Yes, exactly. This way I can plan accordingly to send future payouts to joint owners in a more timely fashion. Knowing the predicted amount would be nice, but maybe not as useful as just the predicted datetime... I know Beaconcha.in provides this information, but I am not at all familiar with how consensus layer APIs work, so not sure if it's standard or if they have proprietary methods.

> The problem I can think of with this is that it's going to be moving as this constantly depends on the number of active validators and that is a value going up and down.

Yeah, I suppose you might need some concept of a virtual event or such, which naturally behaves very differently than an actual thing that happens.
